snippets/controller.cson

```diff
@@ -0,0 +1,68 @@
+'.source.coffee':
+
+
+  'available in model':
+    'prefix': 'arrayController--help'
+    'body':
+      """
+      ${1:
+        # you can use these snippets in a Model function.
+        #
+        # DS.Model.extend(
+        #   example : att<tab>
+        #    _______________________________________________________________
+        #   | type shorthand -> or long version     ->  press tab to see:   |
+        #   |  -----------   ->  -------------      ->   ------------------ |
+        #   |      iC        ->   itemController    ->   itemController...  |
+        #   |      sP        ->   sortProperties    ->   sortProperties...  |
+        #   |      sA        ->   sortAscending     ->   sortAscending...   |
+        #   |_______________________________________________________________|
+        # )
+        #           *press <Backspace> to remove this quick guide.
+
+      }
+      """
+
+
+  'Ember Object Controller':
+    'prefix': 'objectController'
+    'body':
+      """
+        _export = Ember.ObjectController.extend(
+          ${1:#body}
+        )
+      """
+
+
+
+
+  'Ember Array Controller':
+    'prefix': 'arrayController'
+    'body':
+      """
+        _export = Ember.ArrayController.extend(
```

Comment:
This is way complicated. Can we simplify this, to make it just the bare minimum. It should only contain body